src/node_contextify.cc

```diff
@@ -1224,15 +1224,18 @@ static void MeasureMemory(const FunctionCallbackInfo<Value>& args) {
                                                           args[1].As<Object>());
     CHECK_NOT_NULL(sandbox);
     context = sandbox->context();
-    if (context.IsEmpty()) {  // Not yet fully initilaized
+    if (context.IsEmpty()) {  // Not yet fully initialized
       return;
     }
```

Comment:
```suggestion
    if (context.IsEmpty()) return;  // Not yet fully initialized
```

(Just a tiny stylistic issue but it’s something we’ve been moving towards in newer code)

Reply:
I am actually the one with style `always {} with if` :)

Reply:
Let's adopt the unconfigurable https://standardjs.com so that each developer does not get to make a personal choice.

Reply:
@cclauss We already use eslint for our JS code, but this is about C++ code anyway.

But anyway, this isn’t super important here.

Reply:
> Let's adopt the unconfigurable https://standardjs.com so that each developer does not get to make a personal choice.

This is C++ code though, not JavaScript.

Reply:
This file is cpp, so I don't think this will help.